eng/pipelines/official.yml

```diff
@@ -0,0 +1,77 @@
+# This Yaml Document has been converted by ESAI Yaml Pipeline Conversion Tool.
```

Comment:
Should this file be named runtimelab-official.yml to make it possible to checkin to dotnet/runtime main, like [runtimelab.yml](https://github.com/dotnet/runtime/blob/main/eng/pipelines/runtimelab.yml) is checked in today?

This existing setup is meant to allow every runtimelab project to get the CI and build support automatically.

Reply:
Makes sense. It will still require customization for each branch, since it looks like a lot of the branches have custom build steps (e.g. NativeAOT-LLVM is different from swift)

Reply:
Yes, customization is expected.